<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QDictionary</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qdictionary.qrc" >:/resources/qdictionary.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <widget class="DictionaryWidget" native="1" name="dictionaryWidget" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>482</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAboutQt" />
   </widget>
   <widget class="QMenu" name="menu_Tools" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionShowTrayIcon" />
    <addaction name="actionScan" />
    <addaction name="separator" />
    <addaction name="separator" />
    <addaction name="actionSettings" />
   </widget>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Tools" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <widget class="QDockWidget" name="dockWidget" >
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea|Qt::NoDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Dictionary List</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2" >
    <layout class="QVBoxLayout" >
     <property name="leftMargin" >
      <number>1</number>
     </property>
     <property name="topMargin" >
      <number>1</number>
     </property>
     <property name="rightMargin" >
      <number>1</number>
     </property>
     <property name="bottomMargin" >
      <number>1</number>
     </property>
     <item>
      <widget class="DictionaryTree" name="treeWidget" >
       <column>
        <property name="text" >
         <string>1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionSettings" >
   <property name="text" >
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionShowTrayIcon" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show Tray Icon</string>
   </property>
  </action>
  <action name="actionScan" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Scan</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DictionaryTree</class>
   <extends>QTreeWidget</extends>
   <header>dictionarytree.h</header>
  </customwidget>
  <customwidget>
   <class>DictionaryWidget</class>
   <extends>QWidget</extends>
   <header>dictionarywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qdictionary.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionDictionaryList</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>128</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowTrayIcon</sender>
   <signal>triggered(bool)</signal>
   <receiver>actionScan</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
